README.md

```diff
@@ -28,8 +28,8 @@
 
 ## Features
 
-- 📦 **Small**: Just **1.4 KB** gzipped ([3+ times lighter](#benchmarks) than **color** and **tinycolor2**)
-- 🚀 **Fast**: [2+ times faster](#benchmarks) than **color** and **tinycolor2**
+- 📦 **Small**: Just **1.4 KB** gzipped ([3x+ lighter](#benchmarks) than **color** and **tinycolor2**)
+- 🚀 **Fast**: [2x+ faster](#benchmarks) than **color** and **tinycolor2**
```

Comment:
The mix of the symbols/numerals and words stood out to me. I'm not sold on this change here, but mixing 3, +, and then "times" feels a bit off. Not entirely sure why.

Might just be me, and is super frivolous regardless.